openwisp_firmware_upgrader/apps.py

```diff
@@ -8,8 +8,16 @@ class FirmwareUpdaterConfig(AppConfig):
     label = 'firmware_upgrader'
     verbose_name = _('Firmware Management')
 
+    rest_framework_defaults = {
+        'DEFAULT_AUTHENTICATION_CLASSES': [
+            'openwisp_users.api.authentication.BearerAuthentication',
+        ],
+        'DEFAULT_PERMISSION_CLASSES': ['rest_framework.permissions.IsAuthenticated'],
+    }
+
```

Comment:
this could be something good to move to openwisp-users when the API is active, so we can avoid having to re-define this in every module.